sporco/admm/cbpdnli.py

```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2016-2019 by Brendt Wohlberg <brendt@ieee.org>
+# Copyright (C) 2020 by Frank Cwitkowitz <fcwitkow@ur.rochester.edu>
+#                       Brendt Wohlberg <brendt@ieee.org>
 # All rights reserved. BSD 3-clause License.
 # This file is part of the SPORCO package. Details of the copyright
 # and user license can be found in the 'LICENSE.txt' file distributed
 # with the package.
 
-"""Class for ADMM algorithm for convolutional sparse coding with inhibition terms"""
+"""Class for ADMM algorithm for convolutional sparse coding with lateral
+inhibition terms"""
```

Comment:
This is not a huge deal, but I left out "lateral" on purpose since I added a self inhibition term. Self inhibition was not included in the original paper, so it's fine if we still market the file as pertaining to lateral inhibition, while just supporting this additional capability. On the other hand, we could more generally call the implementation "CSC w/ inhibition terms" and rename everything that suggests only lateral inhibition is supported.